DigitalLearningSolutions.Web.Tests/Controllers/MyAccount/MyAccountControllerTests.cs

```diff
@@ -387,16 +410,89 @@ public void
                         null, // null delegateDetailsData -> delegate account is not updated
                         A<string?>._,
                         A<int>._,
+                        true,
+                        false,
                         true
                     )
                 )
                 .MustHaveHappened();
 
+            A.CallTo(

```

Comment:
I don't think you should test this here. You are testing implicit behaviour (i.e. the primary email happens to have changed), when the test name is `EditDetailsPostSave_for_admin_only_user_with_missing_delegate_answers_doesnt_fail_validation_or_update_delegate` - nothing about whether the email should be sent or not.